tool/build.mjs

```diff
@@ -49,27 +52,18 @@ async function buildCLI() {
 }
 
 async function buildPerf() {
-  await esbuild.build({
-    ...sharedOptions,
-    outfile: 'perf/index.js',
-    entryPoints: ['perf/index.ts'],
-    define: {
-      'process.env.NODE_ENV': '"production"',
-    },
-    sourcemap: true,
-    format: 'esm',
-    minify: true,
-  });
+  // Same as what wev build for the npm package but we turn on sourcemaps.
```

Comment:
I think we should also always publish a private package of replicache with source maps.  It's hard to debug issues in our sample apps (that we suspect may be a replicache bug) without source maps.